site/src/pages/ManagementSettingsPage/OrganizationMembersPage.test.tsx

```diff
@@ -19,9 +19,6 @@ jest.spyOn(console, "error").mockImplementation(() => {});
 
 beforeEach(() => {
 	server.use(
-		http.get("/api/v2/experiments", () => {
-			return HttpResponse.json(["multi-organization"]);
```

Comment:
@aslilac are we guarding on `custom-roles` anywhere?

Reply:
not after #14862